fields.inc.php

```diff
@@ -90,8 +90,12 @@ function _pgbar_add_item_defaults(array $item) {
       ],
       'display' => [
         'template' => '',
+        'autostart' => FALSE,
```

Comment:
Will this change the behaviour for existing field items without this value?

Reply:
Ah, good catch. `TRUE` should be the default I guess... :sweat_smile: